src/entities/direct-thread.entity.ts

```diff
@@ -1,11 +1,14 @@
+/* tslint:disable:semicolon */
```

Comment:
Comment-based rule disabling designed for exceptional situations. If you find yourself writing such rules for whole file - most likely you did something wrong.
In this case it's my fault :) I should change tsling in favor of eslint because tsling is [almost deprecated](https://github.com/palantir/tslint#tslint). I found out about three months ago and since that i stopped following the tslint rules, because i'm going to move to eslint somewhen.
So if you want to - you could set up eslint :)

Reply:
I'm not that familiar with eslint so i'd suggest you to migrate the linting yourself, so you ensure the 'code-quality' you expect. I removed the line (it was just so i has no errors)